lms/djangoapps/django_comment_client/urls.py

```diff
@@ -1,6 +1,7 @@
 from django.conf.urls.defaults import url, patterns, include
 
-urlpatterns = patterns('',  # nopep8
-    url(r'forum/?', include('django_comment_client.forum.urls')),
+urlpatterns = patterns(
+    '',
+    url(r'forum/', include('django_comment_client.forum.urls')),
```

Comment:
When i tried to find the thread url in line [#](https://github.com/edx/edx-platform/blob/muhhshoaib/SOL-106-Implement-Synchronous-Notification-emitters-business-logic-in-Open-edX/lms/djangoapps/django_comment_client/base/views.py#L405) that i had added in the payload of NotificationMessage it gives me the following url `/edX/DemoX/Demo_Course/discussion/forum53c486b035b4437c9197a543371e0f03/threads/54c63b6b56c02c9d25000001` Notice there is not a `/` after the forum. I wasn't getting the actual url and after changing the `RegEx` the reverse was working fine.